### PR TITLE
Docs - Upgrade version and release note

### PR DESCRIPTION
**Pull request by abuccts**
__Description__

Upgrade version and release note.

__Major Revision__

- Upgrade package versions
- Add release note for v0.8.0

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

**Review by yukirora — APPROVED** [review]

File: website/blog/2023-04-10-release-0-8.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+### SuperBench Improvements
+
+- Support SuperBench Executor running on Windows.
+- Remove fixed rccl version in rocm5.1.x docker file.
+- Upgrade networkx version to fix installation compatibility issue.
+- Pin setuptools version to v65.7.0.
+- Limit ansible_runner version for Python 3.6.
+- Support cgroup V2 when read system metrics in monitor.
+- Fix analyzer bug in Python 3.8 due to pandas api change.
+- Collect real-time GPU power in monitor.
```
Comment (cp5555) [inline website/blog/2023-04-10-release-0-8.md:26]: Add `Remove unreachable condition when write host list`

File: website/blog/2023-04-10-release-0-8.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Add HPL Benchmark for HPC Linpack Benchmark.
+- Support flexible warmup and non-random data initialization in cublas-benchmark.
+- Support error tolerance in micro-benchmark for CuDNN function.
+- Add distributed inference benchmark.
+- Support tensor core precisions (e.g., FP8) and batch/shape range in cublaslt gemm.
+
+### Model Benchmark Improvements
+
+- Fix torch.dist init issue with multiple models.
+- Support TE FP8 in BERT/GPT2 model.
```
Comment (cp5555) [inline website/blog/2023-04-10-release-0-8.md:40]: Add `Add num_workers configurable in model benchmark`.

File: website/blog/2023-04-10-release-0-8.md (hunk 1 above)
Comment (cp5555) [inline website/blog/2023-04-10-release-0-8.md:26]: Add `Update to cuda12.1, nccl 2.17.1, hpcx 2.14, and mlc 3.10`

Comment (cp5555) [inline website/blog/2023-04-10-release-0-8.md:26]: Add `Fix wrong unit of cpu-memory-bw-latency in doc`

**Pull request merged**